RevenueCat.xcodeproj/project.pbxproj

```diff
@@ -6742,6 +6742,7 @@
 				ONLY_ACTIVE_ARCH = YES;
 				SDKROOT = iphoneos;
 				SUPPORTS_MACCATALYST = YES;
+				SWIFT_ACTIVE_COMPILATION_CONDITIONS = CUSTOMER_CENTER_ENABLED;
```

Comment:
in order to merge to `main`, we just make a new PR that kills this line. And then we bring it back in the integration branch

Reply:
Note that since the flag is only defined for the Xcode project, SPM and CocoaPods builds will already evaluate it to false and skip Customer Center. Carthage won't, however, since it reads from Xcode, so we should remove it to merge into `main`